config/form-content/bestuursorgaan/form.ttl

```diff
@@ -11,46 +11,44 @@
 @prefix mandaat: <http://data.vlaanderen.be/ns/mandaat#>.
 @prefix besluit: <http://data.vlaanderen.be/ns/besluit#>.
 
-ext:bindingEindeF
-    a form:Field;
-    form:displayType displayTypes:date;
-    sh:group ext:bestuursorgaanPG;
-    sh:name "Binding einde";
-    sh:order 3;
-    sh:path mandaat:bindingEinde.
 ext:naamF
     a form:Field;
     form:displayType displayTypes:defaultInput;
     sh:group ext:bestuursorgaanPG;
     sh:name "Naam";
-    sh:order 2;
+    sh:order 1;
     sh:path skos:prefLabel.
-ext:bindingStartF
-    a form:Field;
-    form:displayType displayTypes:date;
-    sh:group ext:bestuursorgaanPG;
-    sh:name "Binding start";
-    sh:order 4;
-    sh:path mandaat:bindingStart.
 ext:bestuursorgaanClassificatieCodeF
     a form:Field;
     form:displayType displayTypes:conceptSchemeSelector;
     sh:group ext:bestuursorgaanPG;
-    sh:name "Standaard type";
-    sh:order 5;
+    sh:name "Type";
+    sh:order 2;
     sh:path <http://data.vlaanderen.be/ns/besluit#classificatie>;
     form:options """{"conceptScheme":"http://data.vlaanderen.be/id/conceptscheme/BestuursorgaanClassificatieCode","searchEnabled":true}""".
+ext:deactivatedAtF
+    a form:Field;
+    form:displayType displayTypes:archivedInput;
+    sh:group ext:bestuursorgaanPG;
+    sh:name "Actief";
+    sh:order 3;
+    sh:path ext:deactivatedAt.
+ext:hiddenBestuurseenheidF
+    a form:Field;
+    sh:name "Bestuurseenheid [hidden input]";
+    sh:order 5000;
+    sh:path besluit:bestuurt.
 
 ext:bestuursorgaanPG
     a form:PropertyGroup; sh:name "Bestuursorgaan"; sh:order 1.
 
 <http://data.lblod.info/id/lmb/forms/bestuursorgaan>
     a form:Form, form:TopLevelForm;
     form:includes
-        ext:bindingEindeF,
+        ext:deactivatedAtF,
         ext:naamF,
-        ext:bindingStartF,
-        ext:bestuursorgaanClassificatieCodeF;
+        ext:bestuursorgaanClassificatieCodeF,
+				ext:hiddenBestuurseenheidF;
```

Comment:
Absolute nitpick, but might be nice to put the fields in order in the form:includes